node/src/actors/session/mod.rs

```diff
@@ -45,8 +45,8 @@ impl HandshakeFlags {
 
 /// Session representing a TCP connection
 pub struct Session {
-    /// Server socket address (local peer)
-    server_addr: SocketAddr,
+    /// Server public socket address (local peer)
```

Comment:
This comment is not clear enough

```suggestion
    /// Public address of the node (the one used by other peers to connect to ours)
```